src/MediaWiki.ts

```diff
@@ -525,6 +525,7 @@ class MediaWiki {
 
       textDir: textDir as TextDirection,
       langMw,
+      langVar: langMw === 'zh' ? 'zh-cn' : null,
```

Comment:
Is it really ok to do this for all mediawikis in Chinese? I suspect it will be fine for all Wikimedia ones (but I'm not even sure) but I'm concerned about other Mediawikis. Is there a way to check this variant is available? Or should this variant not be available, action=parse will just return "default" result?

Reply:
The variant should generally be available on ZH wikis, and if not MW will follow the fallback chain: `['zh-cn', 'zh-hans', 'zh', 'zh-hant', 'en']`

Forcing Mainland China variant might not be ideal for all wikis, especially Taiwanese wikis. But to solve this better we will need #2399 with a new CLI option.